src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/ov_plugin/core_integration.cpp

```diff
@@ -341,12 +341,12 @@ INSTANTIATE_TEST_SUITE_P(smoke_HETERO_OVClassLoadNetworkWithSecondaryPropertiesT
 // IE Class load and check network with ov::device::properties
 INSTANTIATE_TEST_SUITE_P(smoke_CPU_OVClassLoadNetworkAndCheckWithSecondaryPropertiesTest,
                          OVClassLoadNetworkAndCheckSecondaryPropertiesTest,
-                         ::testing::Combine(::testing::Values("CPU", "MULTI:CPU"),
+                         ::testing::Combine(::testing::Values("CPU"),
```

Comment:
Why do you remove `MULTI`? Could you please explain?

Reply:
@iefode as the review in Arch Forum(CVS-104405), the decision is to align the behaviors of MULTI to AUTO cumulative_throuhgput hint in 23.0 and add some of the behaviors back in 23.1/2, so this test case is not suitable for MULTI now.